### DISCUSSION TITLE
How are Downloads used?

### DISCUSSION THREAD

**Discussion by sanjosanjo** [umbrellaplug/umbrellaplug.github.io#237] [Q&A] [answered]
I'm not clear what it means to enable downloads.  Does this automatically save anything that I stream as a local copy?  Or can I choose which items to download?

**Comment by umbrellaplug** [comment, marked as answer]
Enabling downloads gives you the ability to save files from the source selection screen to your selected folder in downloads.

**Reply by sanjosanjo** [reply]
Thanks!

**Comment by Sn3akySl0th** [comment]
Would it be possible to add a download progress section like Seren has? Somewhere to see the progress of all your downloads so that you can see how fast it is downloading and know when it is complete.

**Reply by umbrellaplug** [reply]
I'm sure it would be possible. It's not something I've considered. Downloads really need a lot of work. I'll add this to list of possible changes when downloads are reworked for downloading packs. It's planned but far out.
